### ISSUE TITLE
Allow predefined options for script command arguments 

### ISSUE THREAD

**Issue by Gomeyy** [raycast/extensions#743]
When using a script command that has arguments, allow a list of options for that argument that gets filtered down when typing. 
<img width="756" alt="Untitled-1" src="https://user-images.githubusercontent.com/22477051/151038007-61014e68-1f12-44f5-9f9a-a8a76e42f9e2.png">
![image](https://user-images.githubusercontent.com/22477051/151037981-46a2b4cc-0222-432b-96c9-183314540955.png)

Raycast version: 1.27.0

**Comment by sxn** [comment]
hi @Gomeyy, thanks for the suggestion! However, as it's not related to an extension or to the extension API, would you mind opening an issue in the [script-commands](https://github.com/raycast/script-commands/issues) repository instead? 😺

**Comment by unnamedd** [comment]
Indeed, it is not part of this repository here. But worth mention, this kind of request would be better done on [Slack](https://raycast.com/community) or using the `Send Feedback` from inside of Raycast.
